release/prepareRelease.sh.cmake

```diff
@@ -234,43 +197,53 @@ case "$SYSTEM_NAME" in
 		mv -v ${DESTDIR}share ${macOSPath}/ || exit 1
 	;;
 	Windows)
-		mingw=/usr
 		lua=""
 		case "$SYSTEM_ARCH" in
 			i686|*86)
-				if [ -d /usr/i686-pc-mingw32 ]; then
-					mingw=/usr/i686-pc-mingw32
-				else
-					mingw=/usr/i686-mingw32
-				fi
 				lua=win32
 			;;
 			x86_64|*64)
-				if [ -d /usr/i686-pc-mingw32 ]; then
-					mingw=/usr/x86_64-pc-mingw32
-				else
-					mingw=/usr/x86_64-mingw32
-				fi
 				lua=win64
 			;;
 		esac
-		
+
 		cp -v ${RTTR_SRCDIR}/contrib/lua/${lua}/lua52.dll ${DESTDIR} || exit 1
-		cp -v ${mingw}/bin/libgcc_s_sjlj-1.dll ${DESTDIR} || exit 1
-		cp -v ${mingw}/bin/libminiupnpc-5.dll ${DESTDIR} || exit 1
-		cp -v ${mingw}/bin/libiconv-2.dll ${DESTDIR} || exit 1
-		cp -v ${mingw}/bin/libintl-8.dll ${DESTDIR} || exit 1
-		cp -v ${mingw}/bin/libogg-0.dll ${DESTDIR} || exit 1
-		cp -v ${mingw}/bin/SDL_mixer.dll ${DESTDIR} || exit 1
-		cp -v ${mingw}/bin/SDL.dll ${DESTDIR} || exit 1
-		cp -v ${mingw}/bin/libvorbis-0.dll ${DESTDIR} || exit 1
-		cp -v ${mingw}/bin/libvorbisfile-3.dll ${DESTDIR} || exit 1
+
+		if [ -f $SYSROOT/bin/libgcc_s_sjlj-1.dll ] ; then
```

Comment:
hier ists doch etwas "komplizierter" da der 32bit mingw compiler die sjlj erzeugt, der 64bit die seh. Beide brauchen jedoch in der neuen Version zwingend die libstdc++